src/main/webapp/app/exam/participate/exam-cover/exam-participation-cover.component.html

```diff
@@ -1,14 +1,19 @@
 <div class="w-100 d-flex" *ngIf="!startView">
     <div class="col-md-3"></div>
     <h2 class="col-md-6" style="text-align: center;">
-        <span>{{
-            'artemisApp.examParticipation.finish'
-                | translate
-                    : {
-                          title: exam.title,
-                          fullName: accountName
-                      }
-        }}</span>
+        <span
+            >{{
+                'artemisApp.examParticipation.finish'
+                    | translate
+                        : {
+                              title: exam.title,
+                              fullName: accountName
```

Comment:
I think you can remove this line now, because the full name is not used in the translation any more

Reply:
oh you are right

Reply:
Removed in 8c9e23b5361e231b4a4b3bb46f62239278c75298